codespell_lib/data/dictionary.txt

```diff
@@ -12614,6 +12621,9 @@ sais->says
 salery->salary
 samll->small
 samller->smaller
+sammon->salmon
+samori->samurai
+samue->same
```

Comment:
Add Samuel as a suggestion, although this feels like deja vu from another PR!

Reply:
Done.